packages/kit/src/vite/index.js

```diff
@@ -194,6 +197,8 @@ function kit() {
 				const warning = warn_overridden_config(config, new_config);
 				if (warning) console.error(warning + '\n');
 
+				await call_config_hooks(svelte_config);
```

Comment:
maybe it would be better to move this call to `configResolved` hook

Reply:
I mean, there will be a race condition, another plugins can register in the kit hooks in their `config` hook, and so will be called on `configResolved` kit plugin hook